fastlane_core/lib/fastlane_core/update_checker/update_checker.rb

```diff
@@ -108,20 +108,41 @@ def self.ios_app_identifier
       nil # we don't want this method to cause a crash
     end
 
+    # (optional) Returns the app identifier for the current tool
+    # supply and screengrab use different param names so we have to special case here
+    def self.android_app_identifier(gem_name)
+      # ARGV example:
+      # ["-a", "com.krausefx.app"]
```

Comment:
`-a` param contains a valid package name for screengrab` but not for `supply`. I'll update the comment